recce/cli.py

```diff
@@ -464,16 +495,28 @@ def server(host, port, lifetime, state_file=None, **kwargs):
             "preview": True,
         }
     elif server_mode == RecceServerMode.read_only:
-        is_review = kwargs["review"] = True
-        is_cloud = kwargs["cloud"] = False
-        cloud_options = None
+        if is_cloud:
+            # recce server --cloud --share-url <share-url> --mode read-only
+            # Use state file stored for the share url
+            #
+            # Read-only mode disable these features
+            # - run query
+            # - use checklist
+            # - share
+            #
+            # Usage:
+            #    Used in cloud managed instance. Launch when user click a share link.
+            is_review = kwargs["review"] = True
```

Comment:
Should we put the `is_review = True` outside the `is_cloud` statement?
Because we can launch `read_only` mode without cloud in shared instance.

Reply:
Because the legacy code in the recce-instance-launcher, we have to force to enter review mode to make it compatible.